app/templates/initiative.html

```diff
@@ -39,6 +39,7 @@ <h2>{{ initiative.name }}</h2>
         {% endif %}
 
         <!-- tags -->
+        {% if initiative.tags.all %}
```

Comment:
Let's use `.exists` instead, same for all the other places where it's a relational field.